healthvault-ios-sdk/Classes/Controllers/MHVBrowserAuthBrokerProtocol.h

```diff
@@ -17,17 +17,35 @@
 // limitations under the License.
 
 #import <Foundation/Foundation.h>
-
+#import <WebKit/WebKit.h>
 @class UIViewController;
 
+typedef void (^MHVSignInCompletion)(NSURL *_Nullable successUrl, NSError *_Nullable error);
+
 NS_ASSUME_NONNULL_BEGIN
 
-@protocol MHVBrowserAuthBrokerProtocol <NSObject>
+@protocol MHVBrowserAuthBrokerProtocol <NSObject, WKNavigationDelegate>
```

Comment:
We should probably move this out of the Controllers directory and put it in the Authentication
directory.